e2e/internal/testhelper/doc.go

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+// Package testhelper contains a collecton of test helper functions that
```

Comment:
```suggestion
// Package testhelper contains a collection of test helper functions that
```

Reply:
This needs to be fixed to pass the lint tests

Reply:
thanks!